java/core/src/main/java/sleeper/core/deploy/LambdaJar.java

```diff
@@ -108,6 +120,20 @@ public String getFilenameFormat() {
         return filenameFormat;
     }
 
+    public String getArtifactId() {
+        return artifactId;
+    }
+
+    /**
+     * This method formats the filename with the supplied version.
```

Comment:
There's a problem at "This method". We should follow the Oracle style guide, see under "Class/interface/field descriptions can omit the subject and simply state the object":

https://www.oracle.com/uk/technical-resources/articles/java/javadoc-tool.html#styleguide
https://github.com/gchq/sleeper/blob/develop/docs/development/conventions.md#style